packages/blade/src/components/Amount/__tests__/Amount.native.test.tsx

```diff
@@ -0,0 +1,187 @@
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render heading-large-bold size Amount', () => {
+    const { toJSON } = renderWithTheme(<Amount size="heading-large-bold" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render heading-small size Amount', () => {
+    const { toJSON } = renderWithTheme(<Amount size="heading-small" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render heading-small-bold size Amount', () => {
+    const { toJSON } = renderWithTheme(<Amount size="heading-small-bold" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render medium size Amount', () => {
+    const { toJSON } = renderWithTheme(<Amount size="title-medium" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render medium size Amount', () => {
+    const { toJSON } = renderWithTheme(<Amount size="title-small" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render amount with Decimal value', () => {
+    const { toJSON } = renderWithTheme(
+      <Amount size="heading-small" suffix="decimals" value={1000.22} />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render amount with Humanize value', () => {
+    const { toJSON } = renderWithTheme(
+      <Amount size="heading-small" suffix="humanize" value={1000.22} />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render positive variant Amount ', () => {
+    const { toJSON } = renderWithTheme(<Amount intent="positive" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render information variant Amount ', () => {
+    const { toJSON } = renderWithTheme(<Amount intent="information" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render negative variant Amount ', () => {
+    const { toJSON } = renderWithTheme(<Amount intent="negative" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render neutral variant Amount ', () => {
+    const { toJSON } = renderWithTheme(<Amount intent="neutral" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render positive variant Amount ', () => {
+    const { toJSON } = renderWithTheme(<Amount intent="notice" value={1000} />);
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render positive variant Amount with bold and isAffixSubtle false', () => {
+    const { toJSON } = renderWithTheme(
+      <Amount isAffixSubtle={false} intent="positive" value={1000} />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render information variant Amount ', () => {
+    const { toJSON } = renderWithTheme(
+      <Amount isAffixSubtle={false} intent="information" value={1000} />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render negative variant Amount ', () => {
```

Comment:
I think lets just add test for 1 type of intent and affix as false instead of all of them, snapshots are way too much otherwise and if one is correct, others would be correct too since we're using feedback colors directly